Subject: [patch] skas update 2.6.10.patch
Patch-mainline: never
From: Paolo Giarrusso <blaisorblade_personal@yahoo.it>

This is simply a resync with this change:

http://linux.bkbits.net:8080/linux-2.5/cset@4176817885Kn6zGc09K1fOrpbsJJQA

Since we create an mm and added it to the list, the patch must be updated now,
otherwise it does not even build.

Please, test the resulting kernel and post result with it (I've not yet done
this, in fact you cannot find it on my site).

Signed-off-by: Paolo 'Blaisorblade' Giarrusso <blaisorblade_spam@yahoo.it>
Acked-by: Gerd Knorr <kraxel@suse.de>
---
 mm/proc_mm.c |    5 -----
 1 files changed, 5 deletions(-)

diff -puN mm/proc_mm.c~skas-update-2.6.10 mm/proc_mm.c
--- linux-2.6.10-rc/mm/proc_mm.c~skas-update-2.6.10	2004-11-19 02:03:04.000000000 +0100
+++ linux-2.6.10-rc-paolo/mm/proc_mm.c	2004-11-19 02:07:09.000000000 +0100
@@ -127,11 +127,6 @@ static int open_proc_mm(struct inode *in
 	init_new_empty_context(mm);
 	arch_pick_mmap_layout(mm);
 
-	spin_lock(&mmlist_lock);
-	list_add(&mm->mmlist, &current->mm->mmlist);
-	mmlist_nr++;
-	spin_unlock(&mmlist_lock);
-
 	file->private_data = mm;
 
 	return(0);
_
